env.js

```diff
@@ -27,6 +27,10 @@ const STATUS_URL = process.env.STATUS_URL;
 // This should only be used when Admin is also in unsecured mode.
 const DISABLE_AUTH = process.env.DISABLE_AUTH;
 
+// Configure Google Analytics
+const DISABLE_GA = process.env.DISABLE_GA;
+const GA_TRACKING_ID = process.env.GA_TRACKING_ID;
```

Comment:
We want this to default to `G-0QW4DJWJ20`. so `process.env.GA_TRACKING_ID || 'G-0QW4DJWJ20'